web_poet/example.py

```diff
@@ -0,0 +1,100 @@
+"""This is a simplified, incomplete example of a web-poet framework, written as
+support material for the web-poet tutorial.
+
+No part of this module is intended for production use, and it may change in a
+backward-incompatible way at any point in the future.
+"""
+
+from asyncio import run
+from typing import Any, Dict, Optional, Type
+from warnings import warn
+
+import andi
+import requests
+from url_matcher import URLMatcher
+
+from . import default_registry
+from .page_inputs import HttpClient, HttpResponse, PageParams
+from .pages import ItemPage, is_injectable
+from .utils import ensure_awaitable
+
+warn(
+    (
+        "You should only be importing web_poet.example to follow the web-poet "
+        "tutorial, never as part of production code."
+    ),
+    UserWarning,
+    stacklevel=2,
+)
+
+
+class _HttpClient:
+    async def get(self, url: str) -> HttpResponse:
+        return _get_http_response(url)
+
```

Comment:
Let's have codecov ignore this file so it wouldn't affect the coverage values. 

`.codecov.yml`:
```
ignore:
  - "web_poet/example.py"
```

Ref: https://docs.codecov.com/docs/ignoring-paths